### PR TITLE
Giving the possibility to provide a mapping table name / delta column if

### PR DESCRIPTION
**Pull request by mmenestret**
Giving the possibility to provide a mapping table name / delta column if a single default col is not enough.

**PR Type: Feature**

**Status: Ready to review**

**Breaking change? No**

### CODE REVIEW

File: src/main/scala/com/ebiznext/comet/database/extractor/ExtractScriptGen.scala [hunk 1]
```diff
@@ -30,15 +31,18 @@ object ScriptGen extends StrictLogging {
...
+    * @param deltaColumns Mapping table name -> delta column, has precedence over `defaultDeltaColumn`
     * @return The list of produced files
     */
   def generate(
     domain: Domain,
     scriptTemplateFile: File,
     scriptsOutputPath: File,
-    deltaColumn: Option[String]
+    defaultDeltaColumn: Option[String],
+    deltaColumns: Map[String, String]
```
Comment (AmineSagaama) [inline src/main/scala/com/ebiznext/comet/database/extractor/ExtractScriptGen.scala:43]: Is it enough to have only one delta column for each Table ? Maybe some Tables need a list of columns to apply a diff ?

Comment (hayssams) [reply src/main/scala/com/ebiznext/comet/database/extractor/ExtractScriptGen.scala:43]: I do not yet understand this PR 
Being in front of the beach does not help
But usually it’s better to allow the user to use multiple columns

**Comment by cchepelov** [comment]
Meta-review: why are you staring at PRs while at the beach? 😜

**Comment by hayssams** [comment]
image

Because drinking wine and eating oysters does not make you clever

**Comment by AmineSagaama** [comment]
@mmenestret 
Sounds good to me, to manage multiple columns in another PR.

**Pull request merged**